Makefile

```diff
@@ -26,6 +26,7 @@ install:
 	@echo "Installing cron job..."
 	@install -Dm0644 src/recap.cron $(DESTDIR)$(CRONDIR)/recap
 	@echo "Installing docs..."
+	@install -dm0755 $(DESTDIR)$(DOCDIR)/recap
```

Comment:
I thought so too, but it's failing to do so:
```
# echo $(cat /etc/issue; uname -r)
Ubuntu 14.04.4 LTS \n \l 3.13.0-95-generic

# make install
Installing scripts...
Installing man pages...
Installing configuration...
Installing cron job...
Installing docs...
install: failed to access ‘/usr/local/share/doc/recap/’: No such file or directory
make: *** [install] Error 1
```
It was only fixed with that addition